dashboard/app/models/concerns/curriculum/assignable_course.rb

```diff
@@ -0,0 +1,15 @@
+# Concern to determine if a course version is assignable for a user.
+#
+# To use, include in a model and call the desired method.
+module Curriculum::AssignableCourse
+  extend ActiveSupport::Concern
+
+  def course_assignable?(user)
+    return false unless can_be_instructor?(user)
+    return true if launched?
+    return true if Script.has_any_pilot_access?(user) && has_pilot_access?(user)
+    return true if user.permission?(UserPermission::LEVELBUILDER) && in_development?
```

Comment:
Do we need the `in_development?` conditional here?

Reply:
On further thought, I can answer my own question. But my follow-up is what courses can a levelbuilder be an instructor of? Is it possible we don't make it here because `can_be_instructor?` returns false?

Reply:
Yes - It allows levelbuilders to assign courses that are in development which is matches existing behavior

Reply:
if I am reading correctly, then removing the `&& in_development?` part of the conditional would allow levelbuilders to also assign scripts that are in beta. is there a reason that levelbuilders should be able to assign in_development but not beta? I'm not 100% opposed to this if there is a reason, but if not, then it seems like it would be simpler to allow levelbuilders to assign anything by removing the `&& in_development?` clause. (it could be in another PR)

Reply:
Oh I see what you are suggesting. That makes sense. I will update!